docs/concepts/what-is-an-activity.md

```diff
@@ -26,10 +26,15 @@ In its simplest form, a Temporal Activity is a function or an object method in o
 Temporal does not recover Activity state in case of failures.
 Therefore an Activity function is allowed to contain any code without restrictions.
 
-Activities are invoked asynchronously through task queues.
-A task queue is essentially a queue used to store an Activity task until it is picked up by an available worker.
-The worker processes an Activity by invoking its implementation function.
+#### Invoke Activities through Task Queues
+
+Activities are invoked asynchronously through Task Queues.
+A Task Queue is essentially a queue used to store an Activity task until it is picked up by an available Worker.
```

Comment:
I feel like this sentence can be merged with the one before it, and then you can discuss Workers.